content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,160 @@
+---
+title: TiKV 3.0 GA Release & Benchmarks
+date: 2019-07-08
+---
+
+Today we're very proud to announce the general availability of TiKV 3.0! Before release, TiKV 3.0 underwent a rigorous testing regime, including the [official Jepsen test](https://www.pingcap.com/blog/tidb-passes-jepsen-test-for-snapshot-isolation-and-single-key-linearizability/) with TiDB.
+
+In this version, we tackled many problems of stability at massive scales. Whether it's spanning hundreds of nodes or storing over a trillion key-value pairs, we've seen our users and contributors put TiKV to the test in serious, real world, production scenarios. With 3.0, we've taken our ideas and lessons to bring a host of features that can better support these growing demands.
+
+## Steady at scale
+
+In TiKV 3.0 we've improved our system by:
+
+* **Optimizing the Raft heartbeat mechanism.** With the Hibernate Region feature, TiKV now adjusts the heartbeat frequency according to region activity. This means you'll see less CPU time and network traffic from idle regions.
```

Comment:
```suggestion
* **Optimizing the Raft heartbeat mechanism.** With the [Hibernate Region](https://github.com/tikv/tikv/blob/master/docs/reference/configuration/raftstore-config.md#hibernate-region) feature, TiKV now adjusts the heartbeat frequency according to region activity. This means you'll see less CPU time and network traffic from idle regions.
```

Reply:
Added the link for Hibernate Region

Reply:
@dcalvin That's not a stable link, let's make it https://github.com/tikv/tikv/blob/118f141f69f961e1b0110fa234ffd75c18210dc5/docs/reference/configuration/raftstore-config.md#hibernate-region